src/pages/home/home.module.scss

```diff
@@ -0,0 +1,53 @@
+body, h1, h2, h3, h4, h5, h6, p, blockquote, pre, dl, dd, ol, ul, figure, hr {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Arial, sans-serif;
+}
+
+
+*, ::before, ::after {
+  box-sizing: border-box;
+}
+
+.Home {
+  display: grid;
+  color: #f2f2f2;
+}
+
+.NavBar {
+  display: flex;
+  flex-direction:row;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 50px;
+  background-color: rgb(117, 31, 245);
+  
+}
+
+.Logo {
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+  margin-left: 10px;
```

Comment:
Esse é um dos poucos casos que ta tudo bem usar altura e largura fixa, mas ó, to de olho!